draw/src/main/java/chat/rocket/android/draw/DrawingActivity.kt

```diff
@@ -0,0 +1,185 @@
+package chat.rocket.android.draw
+
+import android.app.Activity
+import android.content.Intent
+import android.content.res.Resources
+import android.graphics.Bitmap
+import android.os.Bundle
+import android.view.View
+import android.widget.SeekBar
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.res.ResourcesCompat
+import androidx.core.view.isVisible
+import kotlinx.android.synthetic.main.activity_drawing.*
+import kotlinx.android.synthetic.main.color_palette_view.*
+import java.io.ByteArrayOutputStream
+
+
+class DrawingActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_drawing)
+
+        image_close_drawing.setOnClickListener {
+            finish()
+        }
+        image_send_drawing.setOnClickListener {
+            val bStream = ByteArrayOutputStream()
+            val bitmap = custom_draw_view.getBitmap()
+            bitmap.compress(Bitmap.CompressFormat.PNG, 100, bStream)
```

Comment:
Compress quality should be 70. Take a look on `compressImage` function from the `ChatRoomPresenter` to see how it works.